EventGenerator/src/PionFilter_module.cc

```diff
@@ -61,23 +66,30 @@ namespace mu2e {
         for(const auto& aParticle : *ah){
           art::Ptr<SimParticle> pp(ah, aParticle.first.asUint());
           float _endglobaltime = pp->endGlobalTime();
-          if( pp->stoppingCode() == ProcessCode::mu2eKillerVolume and std::abs(pp->pdgId()) == PDGCode::pi_plus){
+          if( pp->stoppingCode() == ProcessCode::hBertiniCaptureAtRest and std::abs(pp->pdgId()) == PDGCode::pi_plus){
             const PhysicsParams& gc = *GlobalConstantsHandle<PhysicsParams>();
-            totalweight += exp(-1*pp->endProperTime() / gc.getParticleLifetime(pp->pdgId()));
+            _totalweight += exp(-1*pp->endProperTime() / gc.getParticleLifetime(pp->pdgId()));
+            _ntot += 1;
           }
-          if( pp->stoppingCode() == ProcessCode::mu2eKillerVolume and (std::abs(pp->pdgId()) == PDGCode::pi_plus and _endglobaltime > tmin_ and _endglobaltime < tmax_ )){
+          if( pp->stoppingCode() == ProcessCode::hBertiniCaptureAtRest and (std::abs(pp->pdgId()) == PDGCode::pi_plus and _endglobaltime > tmin_ and _endglobaltime < tmax_ )){
```

Comment:
Was this used before to select pions output from a previous beam stage and then resample the beamline transport simulation? Does this correspond to a change in the production campaign flow? (I genuinely don't know much about how these campaigns are structured, so I'm really just asking out of curiosity!)